test/makefile

```diff
@@ -33,11 +34,27 @@ CFLAGS  += -D_REENTRANT
 CFLAGS  += -I$(MESH_LIB_FOLDER)platform
 
 # Main app
-SOURCES := $(SOURCEPREFIX)test_app.c
+ifeq ($(test_mode),1)
+	SOURCES := $(SOURCEPREFIX)test_mode_app.c
+ifeq ($(rxnode),1)
+	CFLAGS  += -DRXNODE
+endif
+ifeq ($(txnode),1)
+	CFLAGS  += -DTXNODE
+endif
+else
```

Comment:
maybe just a comment to clarify the else // $(test_mode) != 1

Reply:
Will be fixed in next commit. Added indent too to make nested "ifreq" more reabable.